patroni/ha.py

```diff
@@ -491,7 +526,10 @@ def reinitialize(self, cluster):
 
     def process_scheduled_action(self):
         if self.reinitialize_scheduled():
-            if self.cluster.is_unlocked():
+            if self.is_paused():
+                logger.warning('Cluster is in a pause state, can not reinitialize')
```

Comment:
Status of restart and manual failover could be seeing immediately, but reinitialize is considered to be kind of "scheduled" action (it is executed asynchronously, from the main loop).

Hmm, this brings the question, why not to trigger reinitialize directly from API? It will remove some duplicated logic from `Ha`.